gst/milestonexprotect/meson.build

```diff
@@ -0,0 +1,35 @@
+python = import('python') # Meson new Python module https://mesonbuild.com/Python-module.html
+
+python3 = python.find_installation('python3')
+# Python 3 required modules
+python3_required_modules = ['gi']
+
+foreach p : python3_required_modules
+  # Source: https://docs.python.org/3/library/importlib.html#checking-if-a-module-can-be-imported
+  script = 'import importlib.util; import sys; exit(1) if importlib.util.find_spec(\''+ p +'\') is None else exit(0)'
+  if run_command(python3, '-c', script, check: false).returncode() != 0
+    error('Required Python3 module \'' + p + '\' not found')
+  endif
+endforeach
```

Comment:
Since Meson 0.51 you can just use the `modules: [...]` kwarg to find_installation.

Reply:
Thank you! Will test and update